integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/core/EDUCoreSender.java

```diff
@@ -36,18 +36,21 @@
     private final StrategyFactory strategyFactory;
     private final NoarkClient mshClient;
     private final ConversationService conversationService;
+    private final EDUCoreFactory eduCoreFactory;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "eduCoreFactory" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)